src/Services/DocumentSign/SignRequest.php

```diff
@@ -95,7 +110,7 @@ public function sign(string $id, string $from_name, string $from_email, string $
             $response = $this->request('post', '/signrequests', [
                 'from_email' => $from_email,
                 'from_email_name' => $from_name,
-                'document' => config('mvc-document-sign.providers.signrequest.subdomain') . '/documents/' . $id . '/',
+                'document' => config('mvc-document-sign.providers.SignRequest.subdomain') . '/documents/' . $id . '/',
```

Comment:
i dont like the look of this change to:

- add class property private $config
- in constructor set property with config mvc-document.sign.providers.SignRequest
- here in the code use Arr::get to get the subdomain config. Doe this for all configs in this file